core/chains/evm/log/helpers_test.go

```diff
@@ -110,15 +110,17 @@ func (c broadcasterHelperCfg) newWithEthClient(t *testing.T, ethClient evmclient
 
 	orm := log.NewORM(c.db, lggr, config, cltest.FixtureChainID)
 	lb := log.NewTestBroadcaster(orm, ethClient, config, lggr, c.highestSeenHead, mailMon)
+	kst := cltest.NewKeyStore(t, c.db, globalConfig)
 
 	cc := evmtest.NewChainSet(t, evmtest.TestChainOpts{
 		Client:         ethClient,
 		GeneralConfig:  globalConfig,
 		DB:             c.db,
+		KeyStore:       kst.Eth(),
```

Comment:
They were happening because I put keystore.enabledAddresses() call in the builder of Txmgr, which was a core startup critical path. And in a lot of tests, the keystore just wasn't setup for fulfilling this method, threw an error, and failed those tests.

Ultimately I just removed calling above method from the construction logic. Instead moved to Start() method of Confirmer/Broadcaster.